build/lib/electron.ts

```diff
@@ -212,7 +212,7 @@ function getElectron(arch: string): () => NodeJS.ReadWriteStream {
 }
 
 async function main(arch = process.arch): Promise<void> {
-	const version = product.electronRepository ? '19.1.11' : util.getElectronVersion();
+	const version = product.electronRepository ? '22.10.0' : util.getElectronVersion();
```

Comment:
Do we need to unwind this?

Reply:
Yup once builds are available, for now it is safe to merge with this version number.